src/sass/mansonry.scss

```diff
@@ -0,0 +1,24 @@
+$gap: 10pt;
+@media screen and (max-width: 768px) {
```

Comment:
wait what's that file they :D ? mansonry and masonry =D ?

Reply:
supprimé